prowlarr/docker-compose.yml

```
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_WHITELIST: "/api/*"
 
   server:
-    image: linuxserver/prowlarr:0.4.4-nightly@sha256:053efaf2ede2a8e85a960f6db0d6a54e2ca29b13910dc3575eb5a86c686ed302
+    image: linuxserver/prowlarr:1.7.4-nightly@sha256:c6c5541b18c41c86c82bd1937d6878f02ffcdeebdf143ac62bbcf0764d2d37f7
```

Thanks @usernameisJim, I see that v1.7.4.3769 is the latest stable release (https://github.com/Prowlarr/Prowlarr/releases/tag/v1.7.4.3769), so we can use `linuxserver/prowlarr:version-1.7.4.3769`. I'm don't know the context for why we chose the `nightly` tag last time, but we can move away from it.

```suggestion
    image: linuxserver/prowlarr:version-1.7.4.3769@sha256:178ca71cf4694c238fd2324a9f3b9be8e8ac5763098bfeb48dfa945b1fd0aa04
```